src/app/app.component.ts

```diff
@@ -22,7 +21,16 @@ export class AppComponent implements OnInit {
     private readonly authentication: AuthenticationService,
     private readonly errors: ErrorService,
     private readonly router: Router
-  ) {}
+  ) {
+    super();
+  }
+
+  /** ngOnDestroy
+  /* call cleanup
+  */
+  public ngOnDestroy(): void {
+    this.cleanup();
+  }
```

Comment:
Is this necessary? The superclass also implements `ngOnDestroy()`. I have also seen this in other subclasses of `SubscriptionMananger`. In fact, this class doesn't even have to implement `OnDestroy` because the superclass already does that, right?

Reply:
Done